string readFile
string writeFile
float32 duration
geometry_msgs/Pose startPoint
geometry_msgs/Pose endPoint
float32 s_dx
float32 s_dy
float32 s_dz
float32 e_dx
float32 e_dy
float32 e_dz

